renderapi/pointmatch.py

```diff
@@ -114,11 +215,19 @@ def get_matches_from_group_to_group(matchCollection, pgroup, qgroup,
         raise RenderError(r.text)
 
 
-def add_merge_collections(request_url, mergeCollections):
-    if mergeCollections is not None:
-        if type(mergeCollections) is list:
+def add_merge_collections(request_url, mcs):

```

Comment:
unrelated to documentation: is there a reason why we explicitly assign the strings for mergecollections rather than let requests handle it?